workflows/chipseq/chipseq-standard.wdl

```diff
@@ -105,15 +107,17 @@ workflow chipseq_standard {
         max_retries=max_retries
     }
 
-    scatter (pair in zip(bam_to_fastqs.read1s, read_groups)){
+    scatter (tuple in zip(zip(bam_to_fastqs.read1s, bam_to_fastqs.read2s), read_groups)){
```

Comment:
Are you positive this works for all combinations of inputs? I thought WDL enforced "every array in a zip must be of equal length". I think this line would break under certain circumstances. Or maybe my understanding of `zip` is wrong.

Reply:
I need to run some additional test data through, but for both single-end and paired-end data so far, that argument works.

Reply:
Can you point me to an input JSON you're using for both PE and SE? I want to walk myself through some concrete examples, because I'm confused how this `zip` here is working. I'm sure once I can see the "filled in" logic it'll be clear.

Reply:
Sure. I just moved these from my home directory. I did a `sed` to update the paths. Let me know if I missed one.

```
/research_jude/rgs01_jude/groups/zhanggrp/projects/SJCloud/common/athrashe/chipseq_workflow/chipseq_input.json
/research_jude/rgs01_jude/groups/zhanggrp/projects/SJCloud/common/athrashe/chipseq_workflow/chipseq_paired_input.json
```